app/facades/menus/new_facade.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Menus
+  class NewFacade
+    def initialize(**params)
+      @menu = params[:menu]
+    end
+
+    def menu
+      @menu ||= Menu.new(meals: new_meals)
+    end
+
+    def menu_saved?
+      menu.persisted?
```

Comment:
use `delegate :persisted?, to :menu, prefix: true` and call menu_persisted? 
no need for this method